### ISSUE TITLE
audiobooks: reading progression indicator / icon on cover image in library?

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#947]
Follow-up to PR https://github.com/readium/readium-desktop/pull/938

**Comment by llemeurfr** [comment]
Currently, we don't have a progression indicator on ebook covers in the library view. I assume that adding a progression indicator for audiobooks should be done at the same time. 
This is an interesting feature but not urgent IMO. 

Let's keep this issue open until we decide to do both.

**Comment by danielweck** [comment]
Note that with audiobooks, the “current reading location” which is used to restore the exact audio file + time stamp when a publication is open, can be used to show a global percentage progress overlay on top of the audiobook cover. all the data is here already.

with “text” books, global position (or global percentage progression) is not implemented yet (i.e. arbitrary sectioning of html reading order chapters, based on number of bytes per file)

**Comment by danielweck** [comment]
also, the most recently recorded “current reading location” for a text book does not necessarily represent the global progress, for example in scholar / scientific papers, or educational textbooks. 
this notion of linear progress only works reliably with fiction novels, and audio books.

**Comment by llemeurfr** [comment]
> “current reading location” for a text book does not necessarily represent the global progress

True. Still, many reading apps give such an information (just checked the Kobo iOS app, where it is expressed as a % ("15% LU")).

So your take is that we have the data for audiobooks, but not ebooks at the moment.  

What we can do:
1/ display the information in the information panel, where screen reader users can find it (a11y by design). Under More Info; label = "Progression", format = something a screen reader can speak (x hours y minutes z seconds). 
2/ later, implement it also as a blue colored progression bar under the cover (on a grey background), as implemented in Easy Reader from Dolphin. But it means an evolution of the design of the library view.

**Comment by panaC** [comment]
i added a progress indicator in publication Info Dialog only for audiobooks

**Comment by danielweck** [comment]
Note: this PR https://github.com/readium/readium-desktop/pull/930
adds progression information in the publication info popup modal dialog.
